### PR TITLE
reset commandIsRunning in finally block

### PR DESCRIPTION
**Pull request by jedwards1211**
right now it doesn't get reset if an error is thrown.  I assume this would be a bug...

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -90,9 +90,9 @@ let executeHooksWithArgs = (hooks = [], args) => {
             } catch (e) {
                 console.error(e.stack)
                 return resolve(e)
+            } finally {
+                commandIsRunning = _commandIsRunning
```
Comment (christian-bromann) [inline index.js]: I believe the finally block won't be executed in an error case either so we should have it before the return block as well

Comment (jedwards1211) [inline index.js]: huh? finally always gets executed, even if the try or catch block returns.

Comment (christian-bromann) [inline index.js]: Indeed, you are right, my bad 😑

Comment (jedwards1211) [inline index.js]:
```js
function foo() {
  try {
    return 5
  } finally {
    console.log('finally')
  }
}

function bar() {
  try {
    throw new Error('test')
  } catch (error) {
    console.error(error)
    return 5 
  } finally {
    console.log('finally')
  }
}

console.log(foo())
console.log(bar())
```
```
> node finally.js
finally
5
[Error: test]
finally
5
```

**Comment by christian-bromann** [comment]
In order to roll it out to the frameworks, we need to bump the wdio-sync version in each on of them and make sure that the unit test suite passes.

**Pull request merged**